src/serialize.jl

```diff
@@ -0,0 +1,18 @@
+const _pickle = PyNULL()
+
+pickle() = _pickle.o == C_NULL ? copy!(_pickle, pyimport(PyCall.pyversion.major ≥ 3 ? "pickle" : "cPickle")) : _pickle
+
+function Base.serialize(s::AbstractSerializer, pyo::PyObject)
+    pyo.o == C_NULL && return invoke(serialize, (AbstractSerializer, Any), s, pyo)
+    Base.serialize_type(s, PyObject)
+    b = PyBuffer(pycall(pickle()["dumps"], PyObject, pyo))
```

Comment:
Wouldn't it be sufficient to do:
```
Base.serialize_type(s, PyObject)
if pyo.o == C_NULL
    serialize(s, pyo.o)
else
   ...
end
```
rather than calling `invoke`?